VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+Validators also play a major role in governance. If a delegator does not vote, it inherits the vote from its validator. This gives validators a major responsibility in the ecosystem.
+
+### What is a validator's commission?
+
+Fee rewards received by a validator’s pool are split between the validator and its delegators. The validator can apply a commission on the part of the fees that goes to its delegators. This commission is set as a percentage. Each validator is free to set its initial commission, maximum daily commission change rate and maximum commission. The Cosmos Hub enforces the parameter set that each validator self defines. These parameters can only be defined when initially declaring candidacy. 
+
+Note that validators cannot set commission on block provisions. This is to prevent a continuous shift in distribution from delegators to validators which would further the concentration of ATOMs and ultimately weaken the security of the system.
+
+### How are block provisions distributed?
+
+Block provisions are distributed proportionally to all validators relative to their total stake. This means that even though each validator gains atoms with each provision, all validators will still maintain equal weight.
+
+Let’s take an example where we have 10 validators with equal staking power and commission rate of 1%. Let’s also assume that the provision for a block is 1000 Atoms and that each validator has 20% of self-bonded Atoms. These tokens do not go directly to the proposer. Instead, they are evenly spread among validators. So now each validator’s pool has 100 Atoms. These 100 Atoms will be distributed according to each participant’s stake:
+* Validator will get 20 Atoms
+* All delegators will get 80 Atoms
+
+Then, each delegator can claim its part of the 80 Atoms in proportion of their stake in the validator’s staking pool.
+
+### How are fees distributed?
+
+Fees are similarly distributed with the exception that the block proposer can get a bonus on the fees of the block it proposes if it includes more than the strict minimum of required precommits.
+
+When a validator is selected to propose the next block, it must include at least +2/3 precommits for the previous block in the form of validators signatures. However, there is an incentive to include more than 2/3 precommits in the form of a bonus. The bonus is linear: it ranges from 1% if the proposer includes 2/3rd precommits (minimum for the block to be valid) to 5% if the proposer includes 100% precommits. Of course the proposer should not wait too long or other validator may timeout and move on to the next proposer. As such, validators have to find a balance between wait time to get the most signatures and risk of losing out on proposing the next block. This mechanism aims to incentivize non-empty block proposals, better networking between validators as well as mitigating censorship.
```

Comment:
`>2/3`

`... of validator signatures.`

Shouldn't it range from 0% - 5%?

`... or it may timeout ...`

`wait-time`

`... as to mitigate ...`

Reply:
I think it's 1% to incentivise production of non-empty blocks, can you confirm @rigelrozanski ?

Reply:
Rest of comments ok